keras_cv/utils/preprocessing.py

```diff
@@ -323,3 +325,40 @@ def check_fill_mode_and_interpolation(fill_mode, interpolation):
             "Unknown `interpolation` {}. Only `nearest` and "
             "`bilinear` are supported.".format(interpolation)
         )
+
+
+def clip_bounding_box(bounding_boxes, bounding_box_format, images):
```

Comment:
lets move this utility to the bounding_box namespace:

```
def clip_to_image(bounding_boxes, images, bounding_box_format):
    ...
```

then users will call it as:

```
keras_cv.bounding_box.clip_to_image()
```

the subject of the call is implied by the namespace.  Thanks!